rules/promotions/elastic_endpoint.toml

```diff
@@ -11,6 +11,7 @@ Generates a detection alert each time an Elastic Endpoint alert is received. Ena
 immediately begin investigating your Elastic Endpoint alerts.
 """
 enabled = true
+exceptions_list = [{ id = 'endpoint_list', namespace_type = 'agnostic', type = 'endpoint' }]
```

Comment:
can you run `toml-lint` to convert and order this

Reply:
There were quite a few other linter fixes by the way -- I only pushed the one related to this rule though:


<details><summary>Other updates from linting:</summary>
<p>

```
	modified:   rules/aws/defense_evasion_config_service_rule_deletion.toml
	modified:   rules/aws/defense_evasion_waf_rule_or_rule_group_deletion.toml
	modified:   rules/aws/exfiltration_ec2_snapshot_change_activity.toml
	modified:   rules/aws/privilege_escalation_root_login_without_mfa.toml
	modified:   rules/linux/defense_evasion_attempt_to_disable_iptables_or_firewall.toml
	modified:   rules/linux/defense_evasion_hex_encoding_or_decoding_activity.toml
	modified:   rules/linux/defense_evasion_hidden_file_dir_tmp.toml
	modified:   rules/linux/defense_evasion_kernel_module_removal.toml
	modified:   rules/linux/discovery_virtual_machine_fingerprinting.toml
	modified:   rules/network/command_and_control_dns_directly_to_the_internet.toml
	modified:   rules/network/command_and_control_ftp_file_transfer_protocol_activity_to_the_internet.toml
	modified:   rules/network/command_and_control_irc_internet_relay_chat_protocol_activity_to_the_internet.toml
	modified:   rules/network/command_and_control_nat_traversal_port_activity.toml
	modified:   rules/network/command_and_control_port_26_activity.toml
	modified:   rules/network/command_and_control_port_8000_activity_to_the_internet.toml
	modified:   rules/network/command_and_control_pptp_point_to_point_tunneling_protocol_activity.toml
	modified:   rules/network/command_and_control_proxy_port_activity_to_the_internet.toml
	modified:   rules/network/command_and_control_rdp_remote_desktop_protocol_from_the_internet.toml
	modified:   rules/network/command_and_control_smtp_to_the_internet.toml
	modified:   rules/network/command_and_control_sql_server_port_activity_to_the_internet.toml
	modified:   rules/network/command_and_control_ssh_secure_shell_from_the_internet.toml
	modified:   rules/network/command_and_control_ssh_secure_shell_to_the_internet.toml
	modified:   rules/network/command_and_control_telnet_port_activity.toml
	modified:   rules/network/command_and_control_tor_activity_to_the_internet.toml
	modified:   rules/network/command_and_control_vnc_virtual_network_computing_from_the_internet.toml
	modified:   rules/network/command_and_control_vnc_virtual_network_computing_to_the_internet.toml
	modified:   rules/network/initial_access_rdp_remote_desktop_protocol_to_the_internet.toml
	modified:   rules/network/initial_access_rpc_remote_procedure_call_from_the_internet.toml
	modified:   rules/network/initial_access_rpc_remote_procedure_call_to_the_internet.toml
	modified:   rules/network/initial_access_smb_windows_file_sharing_activity_to_the_internet.toml
	modified:   rules/okta/credential_access_attempted_bypass_of_okta_mfa.toml
	modified:   rules/okta/impact_attempt_to_revoke_okta_api_token.toml
	modified:   rules/okta/impact_possible_okta_dos_attack.toml
	modified:   rules/okta/initial_access_suspicious_activity_reported_by_okta_user.toml
	modified:   rules/okta/okta_attempt_to_deactivate_okta_mfa_rule.toml
	modified:   rules/okta/okta_attempt_to_delete_okta_policy.toml
	modified:   rules/okta/okta_attempt_to_modify_okta_mfa_rule.toml
	modified:   rules/okta/okta_attempt_to_modify_okta_network_zone.toml
	modified:   rules/okta/okta_attempt_to_modify_okta_policy.toml
	modified:   rules/okta/okta_attempt_to_modify_or_delete_application_sign_on_policy.toml
	modified:   rules/okta/okta_threat_detected_by_okta_threatinsight.toml
	modified:   rules/okta/persistence_administrator_privileges_assigned_to_okta_group.toml
	modified:   rules/okta/persistence_attempt_to_create_okta_api_token.toml
	modified:   rules/okta/persistence_attempt_to_deactivate_mfa_for_okta_user_account.toml
	modified:   rules/okta/persistence_attempt_to_deactivate_okta_policy.toml
	modified:   rules/okta/persistence_attempt_to_reset_mfa_factors_for_okta_user_account.toml
	modified:   rules/promotions/elastic_endpoint.toml
	modified:   rules/promotions/external_alerts.toml
	modified:   rules/windows/defense_evasion_execution_msbuild_started_unusal_process.toml
```
</p>
</details>